### ISSUE TITLE
Time Zone Discrepancy in Sonarr for Episode Release Times

### ISSUE THREAD

**Issue by OMARXOXO** [Sonarr/Sonarr#7179]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Sonarr displays the incorrect episode release time for The Lord of the Rings: The Rings of Power. It shows the release time as 7:00 AM (Asia/Dubai, UTC+4), while the actual release time on Amazon Prime Video is 11:00 AM.

### Expected Behavior

Sonarr should display the correct local release time of 11:00 AM for Dubai, or convert TVDB’s 3:00 AM EST to 12:00 PM (noon), based on the system's correct time zone.

### Steps To Reproduce

1. Set system time zone to Asia/Dubai (UTC+4).
2. Configure Sonarr to monitor The Lord of the Rings: The Rings of Power.
3. Compare Sonarr’s release time to the actual release time on Amazon Prime Video.
4. Sonarr displays 7:00 AM, while the correct time should be 11:00 AM.

### Environment

```markdown
- OS: Synology DSM 7.2.1
- Sonarr: 4.0.9.2244
- Docker Install: Yes
- Using Reverse Proxy: Yes
- Browser: Brave
- Database: SQLite (default)
```

### What branch are you running?

Main

### Trace Logs?

Unfortunately, no trace logs are available because no errors are shown, only incorrect time handling.

### Anything else?

screenshot showing the time discrepancy: [https://imgur.com/BFH6Faq]

This issue may be related to how Sonarr is interpreting episode release times from TVDB. TVDB lists US-based shows in Eastern Standard Time (EST) and has specific rules for streaming services. However, the time zone conversion from EST to Asia/Dubai (UTC+4) doesn’t seem to be happening correctly in Sonarr.

You can find more about TVDB’s guidelines on release times here:

[TVDB Knowledgebase on Time Zones](https://support.thetvdb.com/kb/)
Similar issues regarding incorrect time zone handling have been reported in the past. It appears that Sonarr is not converting the 3:00 AM EST release time from TVDB properly into 12:00 PM (noon) for my local time in Dubai.

Let me know if you need additional logs or information!

**Comment by SpookyRoger** [comment]
I am having the same issue.  I am in NZ and it is off by the same margin.

**Comment by OMARXOXO** [comment]
Thanks for the reply, mynameisbogdan!

Just to clarify, TVDB does provide the air time for certain shows. For example, on the [TVDB page for The Lord of the Rings: The Rings of Power](https://thetvdb.com/series/the-lord-of-the-rings), it clearly lists the air time as Thursday at 3:00 AM EST. This air time is visible on the front page of the series' listing.

The issue I’m experiencing seems to be related to how Sonarr interprets and converts this time into my local time zone (Asia/Dubai). Sonarr shows 7:00 AM, while the actual air time on Amazon Prime Video is 11:00 AM in my region. So, it’s likely that Sonarr is pulling the air time from TVDB but incorrectly converting it to my local time.

Thanks again for helping clear this up!

**Comment by Thirrian** [comment]
Keep in mind TVDB is notorious for not having the same info on their website and their API, which is used by skyhook (layer between TVDB and your sonarr instance).

**Comment by OMARXOXO** [comment]
Thanks for the insight, Thirrian!

Just to make sure I understand correctly, are you saying that the difference between TVDB’s website and their API is due to a persistent discrepancy, where the API might not always have the same level of detail (like air times) as the website? Or are you referring to temporary delays between the two sources?

I want to be sure I’m understanding the root of the issue. Thanks again for your help!

**Comment by markus101** [comment]
Thanks for reporting, this'll be fixed as caches expire. In the future please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.sonarr.tv/), or [IRC](https://web.libera.chat/?channels=#sonarr) to report metadata discrepancies.
